people.json

```diff
@@ -483,20 +483,14 @@
             "Web application",
             "Mobile app development",
             "Desktop app",
-            "Application design",
-            "vis.js",
             "C Sharp (programming language)",
             "JavaScript",
             "ASP.NET",
-            "Microsoft Azure",
-            "Cordova",
             "SQL",
-            "Alfresco",
-            "UWP",
-            "C (programming language)"
-        ],
-        "technologies": [
```

Comment:
Hi Andrew.  Just to confirm that you intended to remove the "technologies" part from your section?

Reply:
Yes, that section isn't relevant any more - I am no longer the contact for FBMH stuff!